GOVERNANCE.md

```diff
@@ -8,31 +8,87 @@ This document defines the project governance for Contour.
+* Expected to be responsive to review requests
+* Assigned PRs to review related to subproject of expertise
+* Assigned test bugs related to subproject of expertise
 
 ### Maintainers
 
 New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.
 
+#### Maintainer Expectations
+
+Maintainers have the ability to merge code into the project. Anyone can
+become a Container maintainer (see "Becoming a maintainer" below.)
+
+As such, there are certain expectations for maintainers. Contour maintainers
+are expected to:
+
+* Review pull requests, triage issues, and fix bugs in their areas of expertise, ensuring that all changes go through the project's code review and integration processes.
+* Monitor cncf-contour-* emails and the Contour Slack, and help out when possible.
+* Rapidly respond to any time-sensitive security release processes.
+
+Some Maintainers are responsible for one or more components within a project, acting as technical leads for that component. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
+
+If a maintainer is no longer interested in or cannot perform the duties
+listed above, they should move themselves to emeritus status. If necessary,
+this can also occur through the decision-making process outlined below.
+
+### SIGs and SIG-owner
+
+SIGs oversee and coordinate the interests and needs of end users and projects in a particular area. By definition, they are long-lived groups that coordinate their work with the maintainers team and are led primarily by recognized experts in the relevant field and supported by other contributors.
+
 ### Supermajority
 
 A supermajority is defined as two-thirds of members in the group.
 A supermajority of [Maintainers](#maintainers) is required for certain
 decisions as outlined above. Voting on decisions can happen on the mailing list, GitHub, Slack, email, or via a voting service, when appropriate. Maintainers can either vote "agree, yes, +1", "disagree, no, -1", or "abstain". A vote passes when supermajority is met. An abstain vote equals not voting at all.
 
+
+### Becoming a maintainer
+
+* Anyone can become a Contour maintainer
+* Maintainers should be proficient in Go
+* have relevant domain expertise
+* have the time and ability to meet the maintainer expectations above
+* and demonstrate the ability to work with the existing maintainers and project processes
+
+*New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the maintainers.*
```

Comment:
Should we move this under the Maintainers section, above?

Reply:
Yeah, feels like this should be a subsection of the Maintainers heading.